### PR TITLE
Feature/even more php fixes

### PR DESCRIPTION
**Pull request by quartje**
Uninstall php56 completely, and manually install the remi repo file. Composer is eleminated as well.

### CODE REVIEW

File: roles/php/tasks/main.yml [hunk 1]
```diff
@@ -1,23 +1,24 @@
-# Install the "REMI" repo which contain newer php packages that override the default
-# that come with the distro (CentOS7: 5.4; CentOS6: 5.3)
-- name: check for presence of the remi repository file
-  stat:
-    path: "/etc/yum.repos.d/remi.repo"
```
Comment (thijskh) [inline roles/php/tasks/main.yml]: Is it now certain that this file is gone?

File: roles/php/tasks/main.yml [hunk 2]
```diff
@@ -30,9 +31,15 @@
       - php72-php-soap
       - php72-php-xml
       - php72-php-gd
-      - composer
     state: present
 
+- name: Remove all php56 packages and composer
+  yum:
```
Comment (thijskh) [inline roles/php/tasks/main.yml]: I put removal at the end so the install is not needlessly broken while executing the task. Remove 5.6 when 7.2 is fully configured

File: roles/php/tasks/main.yml (hunk 1 above)
Comment (quartje) [reply roles/php/tasks/main.yml]: yes, it's in the remi-release package, which is being removed. Yum saves a copy in /etc/yum.repos.d/remi.repo.rpmsave

File: roles/php/tasks/main.yml (hunk 2 above)
Comment (quartje) [reply roles/php/tasks/main.yml]: I've added a commit to move it to the end

**Pull request merged**